biorxiv/journal_tracker/map_preprints_to_journals.py

```diff
@@ -0,0 +1,203 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# # Following the Preprint to Published Path
+
+# The goal of this notebook is to map preprint dois to published dois and published dois to Pubmed Central articles.
+
+# In[1]:
+
+
+from habanero import Crossref
+import pandas as pd
+from ratelimit import limits, sleep_and_retry
+import tqdm
+from urllib.error import HTTPError
+
+
+# In[2]:
+
+
+preprints_df = pd.read_csv("../exploratory_data_analysis/output/biorxiv_article_metadata.tsv", sep="\t")
+preprints_df.head()
+
+
+# In[3]:
+
+
+dois = (
+    preprints_df
+    .doi
+    .unique()
+)
+print(len(dois))
+
+
+# In[4]:
+
+
+cf = Crossref(mailto="dnicholson329@gmail.com")
```

Comment:
You may want to load your email from a config file or something. If you end up with a lot of people running this code, they may get blocked or rate limited for all using the same email

Reply:
Fantastic catch. Yeah my email should not be there. I definitely __don't__ want people running my code and making me be responsible for their foolishness.